templates/taxi/driver_list.html

```diff
@@ -7,7 +7,11 @@
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if user.id == driver.id %}
```

Comment:
The condition `user.id == driver.id` is used to identify the logged-in user. Ensure that the `user` context variable is available in the template, typically provided by Django's authentication context processor.